.github/workflows/CI.yml

```diff
@@ -49,7 +47,7 @@ jobs:
 
       - name: 'Get Previous tag'
         id: get_previous_tag
-        uses: "WyriHaximus/github-action-get-previous-tag@v1.0.1"
+        uses: "WyriHaximus/github-action-get-previous-tag@v1.1"
```

Comment:
The tag format `v1.1` is not a convention that is generally used by GH actions. I suggest to either use `v1` or a full tag like `v1.1.0`